content/22.vaccines.md

```diff
@@ -171,7 +171,66 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
 
 #### COVID-19 Vaccine AstraZeneca
 
-#### Sputnik-V
+#### Sputnik-V (Gam-COVID-Vac and Gam-COVID-Vac-Lyo)
+
+The vaccine Gam-COVID-Vac, nicknamed Sputnik V in reference to the space race, was developed by the Gamaleya National Center of Epidemiology and Microbiology in Moscow. 
+Gamaleya is an organization with prior experience using the adenovirus platform for the development of vaccines for the Middle East respiratory system and Ebola virus, although neither of the previous vaccines were internationally licensed [@url:https://www.who.int/immunization/sage/meetings/2018/october/2_Ebola_SAGE2018Oct_BgDoc_20180919.pdf].
+The development of Sputnik V was financed by the Russian Direct Investment Fund [@doi:ft7j; @doi:ghs3sn]. 
```

Comment:
```suggestion
The development of Sputnik V was financed by the Russian Direct Investment Fund [@doi:10/ft7j; @doi:10/ghs3sn]. 
```
Thank you for doing the short dois! Their syntax is tricky.

Reply:
oops... it is always the lancet journals have these weird dois